utils/consts.go

```diff
@@ -562,14 +562,14 @@ const (
 	MetaSubscribers                 = "*Subscribers"
 	MetaDerivedChargersV            = "*DerivedChargers"
 	MetaTpRatingPlans               = "*TpRatingPlans"
-	MetaTpLcrs                      = "*TpLcrs"
+	MetaTpLCRRules                  = "*TpLcrs"
```

Comment:
all meta instructions need to be written as commands (ie here we should have *tp_lcrs